x/liquidity/spec/02_state.md

```diff
@@ -1,46 +1,52 @@
-<!--
-order: 2
--->
+<!-- order: 2 -->
 
-# State
+ # State
+
+The Liquidity Module keeps track of the Pool and PoolBatch states. The state represents your app at a given moment.
 
 ## Pool
 
-`Pool` defines the liquidity pool that contains pool information
+The Pool state contains information about the pool type, reserve coin denoms, reserve address, and denom of this liquidity pool.
+
+The Pool state stores static information of a liquidity pool in this structure:
 
 ```go
 type Pool struct {
-	Id                     uint64         // index of this liquidity pool
-	TypeId                 uint32         // pool type of this liquidity pool
-	ReserveCoinDenoms      []string       // list of reserve coin denoms for this liquidity pool
-	ReserveAccountAddress  string         // reserve account address for this liquidity pool to store reserve coins
-	PoolCoinDenom          string         // denom of pool coin for this liquidity pool
+    Id                     uint64         // index of this liquidity pool
```

Comment:
@dongsam you'll see that I updated the comments for the message code, can we update the code itself? lmk how I can help

Reply:
Of course, I'll apply it all at once just before merging this PR